### PR TITLE
feat - Refactor layout and implementing styled component

### PR DESCRIPTION
**Pull request by louisgv**
+ Adding styled component and jazz

+ Remove splitpane and implement our own layout

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 
 import {
-    BrowserRouter as Router,
-    Route,
+  BrowserRouter as Router,
+  Route
 } from 'react-router-dom';
 
+
+import './index.css';
```
Comment (ColdSauce) [inline src/index.js:13]: Should we stick with styled components as a standard for this project?

File: src/index.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+
 const App = () => (
-  <Router basename="/">
-    <Route exact path="/" component={Messenger} />
+  <Router basename="/commit">
```
Comment (ColdSauce) [inline src/index.js:16]: Why have this be the base?

**Review by ColdSauce — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 2 above)
Comment (louisgv) [reply src/index.js:16]: Oh, if we are gonna expose this via github, we need the base path the same as the name of the repo.

File: src/index.js (hunk 1 above)
Comment (louisgv) [reply src/index.js:13]: We should use both. In this case the global css is used to style body and generic tags like h1, h2 etc... Styled component is for per component styling.

Comment (ColdSauce) [reply src/index.js:13]: k

File: src/index.js (hunk 2 above)
Comment (ColdSauce) [reply src/index.js:16]: kk

**Review by ColdSauce — APPROVED** [review]

**Pull request merged**